widget-reference/iframe.md

```diff
@@ -0,0 +1,22 @@
+---
+description: >-
+  Iframe widget is used to display iframes in your app.
+---
+
+# Iframe
+
+## Image/gif of the widget on the canvas with the icon of the widget in the sidebar
+
+## Properties
+| Property      | Description           |
+| :------------ | :-------------------- |
+| **source** | Sets the URL of the page to embed. |
+| **title** | Labels the content of the page to embed. |
+| **borderColor** | Sets the color of the border surrounding the page to embed. |
+| **borderOpacity** | Sets the color opacity of the border surrounding the page to embed. |
+| **borderWidth** | Sets the width of the border surrounding the page to embed. |
+
+| Action      | Description         |
+| :---------- | :------------------ |
+| **onURLChanged** | Sets the action to be run when the source url is changed. |
+| **onMessageReceived** | Sets the action to be run when a message event is received from the embedded page. |
```

Comment:
This should be `Sets the action to be run when a postMessage event is received from the embedded page`

Reply:
@wmdev0808 please check the above!